<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastBtn</class>
 <widget class="QWidget" name="ForecastBtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>124</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="weekLab">
     <property name="text">
      <string>星期</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="weatherLab">
     <property name="text">
      <string>天气</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tempLab">
     <property name="text">
      <string>温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
